vars/archlinux.yml

```diff
@@ -0,0 +1,3 @@
+---
+locale_conf_path: /etc/locale.conf
+...
```

Comment:
Please add an empty line before all `...`. Unfortunately, I haven't finished [adding this to yamllint](https://github.com/adrienverge/yamllint/pull/393), but that's one check that's missing :smile: